src/main/java/nl/novi/techiteasy1121/controllers/TelevisionController.java

```diff
@@ -43,52 +46,63 @@ public ResponseEntity<List<TelevisionDto>> getAllTelevisions(@RequestParam(value
 
     }
 
-    // De return waarde is ook hier een TelevisionDto in plaats van een Television
     @GetMapping("/televisions/{id}")
     public ResponseEntity<TelevisionDto> getTelevision(@PathVariable("id")Long id) {
 
-        // We spreken hier ook weer een service methode aan in plaats van direct de repository aan te spreken
         TelevisionDto television = televisionService.getTelevisionById(id);
 
             return ResponseEntity.ok().body(television);
 
     }
 
-    // Ook hier returnen we weer een TelevisionDto, maar ook de parameter is een dto geworden.
-    // Het is niet verplicht om een "outputdto" en een "inputdto" te hebben, zeker omdat ze in dit geval hetzelfde zijn,
-    // maar we willen jullie laten zien dat het mogelijk is. In sommige gevallen kan het zelfs nodig zijn.
     @PostMapping("/televisions")
-    public ResponseEntity<TelevisionDto> addTelevision(@Valid @RequestBody TelevisionInputDto televisionInputDto) {
+    public ResponseEntity<Object> addTelevision(@Valid @RequestBody TelevisionInputDto televisionInputDto) {
 
-        // Hier gebruiken we weer een service methode in plaats van direct de repository aan te spreken.
         TelevisionDto dto = televisionService.addTelevision(televisionInputDto);
 
         return ResponseEntity.created(null).body(dto);
 
     }
 
-    // Hier veranderd niks aan de methode. We hebben niet meer de naam van de pathvariabele expliciet genoemd, omdat de
-    // parameter-naam overeen komt met de naam van de pathvariabele.
     @DeleteMapping("/televisions/{id}")
     public ResponseEntity<Object> deleteTelevision(@PathVariable Long id) {
 
-        // Hier gebruiken we weer een service methode in plaats van direct de repository aan te spreken.
         televisionService.deleteTelevision(id);
 
         return ResponseEntity.noContent().build();
 
     }
 
-    // Deze methode returned nu een ResponseEntity<TelevisionDto> in plaats van een ResponseEntity<Television> en deze
-    // methode vraagt nu om een Long en een TelevisionInputDto in de parameters in plaats van een Long en een Television.
     @PutMapping("/televisions/{id}")
-    public ResponseEntity<TelevisionDto> updateTelevision(@PathVariable Long id, @Valid @RequestBody TelevisionInputDto newTelevision) {
+    public ResponseEntity<Object> updateTelevision(@PathVariable Long id, @Valid @RequestBody TelevisionInputDto newTelevision) {
 
-        // Hier gebruiken we weer een service methode in plaats van direct de repository aan te spreken.
-        // Alle logica die hier eerst stond, is nu ook verplaatst naar de service laag.
         TelevisionDto dto = televisionService.updateTelevision(id, newTelevision);
 
         return ResponseEntity.ok().body(dto);
     }
 
+    // Onderstaande 2 methodes zijn endpoints om andere entiteiten toe te voegen aan de Television.
+    // Dit is één manier om dit te doen, met één PathVariable en één RequestBody.
+    @PutMapping("/televisions/{id}/remotecontroller/{remoteId}")
```

Comment:
Zorg ervoor dat je de juiste REST-principes volgt in je endpoints, bijvoorbeeld door de juiste HTTP-methoden te gebruiken en duidelijke paden te definiëren.